src/services/make.service.ts

```diff
@@ -19,3 +20,31 @@ export async function runSavePDFScenario() {
     return { error };
   }
 }
+
+export async function sendMessageWebhook(bookingInfo: Booking, boatDetails: Boat) {
+  try {
+    const queryParams = new URLSearchParams({
+      date: moment(bookingInfo.Date).format("DD/MM/YY"),
+      id: bookingInfo.id,
+      firstName:bookingInfo["First Name"],
+      lastName:bookingInfo["Last Name"],
+      customerEmail: bookingInfo.Email,
+      boatName: boatDetails.Nombre,
+      pricePerMile: (boatDetails.MilePrice ?? "0").toString(),
+      totalPassengers: (bookingInfo["Total Passengers"] ?? "0").toString(),
+    }).toString();
+    
+    const res = await fetch(`https://hook.eu2.make.com/mkshx7rk3edsbzg55lty4ej77fwiukg1?${queryParams}`, {
```

Comment:
These hardcoded variables from 3erd parties will be great to have them in a constant file.